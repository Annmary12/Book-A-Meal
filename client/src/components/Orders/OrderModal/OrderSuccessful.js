import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import checkMark from '../../../../assets/img/check-mark.svg';
import orderStyles from './order-modal.scss';

const OrderSuccessful = ({ handleClose }) => (
  <div style={{ textAlign: 'center' }}>
    <h3>Order placed successfully</h3>
    <img src={checkMark} alt="check-mark" width="50" />
    <p>
      <Link to="/orders">View order history</Link>
    </p>
    <p>
      Please note that order update and cancellation
      can only be done 15 minutes after the order is placed.
    </p>

    <div style={{ paddingTop: '35px' }}>
      <button
        className={orderStyles.closeModalBtn}
        onClick={handleClose}
      >
      close
      </button>
    </div>
  </div>
);

OrderSuccessful.propTypes = {
  handleClose: PropTypes.func.isRequired
};

export default OrderSuccessful;
