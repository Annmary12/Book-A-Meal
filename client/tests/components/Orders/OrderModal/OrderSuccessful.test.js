import React from 'react';
import OrderSuccessful from '../../../../src/components/Orders/OrderModal/OrderSuccessful';

describe('OrderSuccessful component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <OrderSuccessful
        handleClose={() => {}}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});