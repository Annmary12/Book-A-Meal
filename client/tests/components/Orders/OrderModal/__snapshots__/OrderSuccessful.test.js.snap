// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderSuccessful component should match snapshot 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h3>
    Order placed successfully
  </h3>
  <img
    alt="check-mark"
    src="test-file-stub"
    width="50"
  />
  <p>
    <Link
      replace={false}
      to="/orders"
    >
      View order history
    </Link>
  </p>
  <p>
    Please note that order update and cancellation can only be done 15 minutes after the order is placed.
  </p>
  <div
    style={
      Object {
        "paddingTop": "35px",
      }
    }
  >
    <button
      className="closeModalBtn"
      onClick={[Function]}
    >
      close
    </button>
  </div>
</div>
`;
